src/cli/tests/cli.spec.ts

```diff
@@ -10,7 +13,7 @@ function cli(args, cwd): Promise<{ code: number; error: ExecException; stdout: s
 			(error, stdout, stderr) => {
 				resolve({
 					code: error && error.code ? error.code : 0,
-					error,
+					error: error as ExecException,
```

Comment:
is that necessary?

Reply:
yes, else it throws this error 
type 'ExecException | null' is not assignable to type 'ExecException'.
  Type 'null' is not assignable to type 'ExecException'.ts(2322)

Reply:
In that case you shouldn't cast it to this type but instead update the return type to include null